index.rst

```diff
@@ -44,14 +44,636 @@
+It is currently available at the `summit <http://amor01.cp.lsst.org/>`_ (VPN required) and all other test stands.
+
+The system is developed using the popular React JavaScript framework for the frontend, with Python in the backend.
+It contains an editing tool that allow users to create and customize views, but this feature relies exclusively on existing widget-like components.
+Extending the system beyond the vendor-provided "widgets" would require some knowledge of React and JavaScript, which is not common in the project.
+Nevertheless, LOVE does contain a special widget that allows us to embed other web pages into a particular view. 
+This could be used, for instance, to embed `Bokeh Plotting Applications`_ alongside other LOVE components.
+
+
+
+Nublado (Jupyter Notebook) Interface
+''''''''''''''''''''''''''''''''''''
+
+The summit `Nublado interface <https://summit-lsp.lsst.codes/>`_ (VPN required) provides the user with a Jupyter Lab interface and the required libraries/packages to perform standard observatory operations including sending commands and running SAL scripts.
+It can also be used to query the EFD.
+This tool is set up to mimic the `RSP`_ and test-stand environments to the maximum extent possible, providing all the functionalities of a Jupyter Notebook but with direct access to the control system.
+Observers are expected to use this tool to perform commands or sequences that are not encapsulated into a SAL script.
+The Nublado interface is also useful for on-the-fly analysis and/or custom monitoring.
+
+
+Camera Visualization Tool
+''''''''''''''''''''''''''
+The camera image visualization tool is designed to run on the camera diagnostic cluster and to deliver full-resolution focal plane images to the operator within a few seconds of image acquisition. 
+The front-end is web based and supports natural zooming and panning, including zooming to full pixel resolution, as well as the ability to display pixel coordinates and raw pixel value by moving the mouse over the image. 
+It supports a simple but extensible set of image processing algorithms which have been found to be useful for viewing images during camera testing.
+
+The tool uses the `International Image Interchange Framework <https://iiif.io/>`_ (IIIF) with the front-end based on the open source `OpenSeadragon <https://openseadragon.github.io/>`_ framework, and the backend based on the open source `Cantaloupe <https://cantaloupe-project.github.io/>`_ product, modified to be able to read images from  FITS files and directly from the DAQ.
+The tool is currently deployed at SLAC for camera construction, and on the summit in Chile for Auxtel and ComCam.
+
+
+Engineering Facilities Database and Large File Annex
+'''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The `Engineering Facilities Database (EFD) <https://sqr-034.lsst.io/#introduction>`_ records all commands, events, and telemetry sent over the DDS control network.
```

Comment:
It also records acknowledgements... Maybe say "all data sent by components over the DDS control network"?